drivers/gpu/drm/vc4/vc4_hdmi.c

```diff
@@ -2281,6 +2289,42 @@ static int vc5_hdmi_init_resources(struct vc4_hdmi *vc4_hdmi)
 		return PTR_ERR(vc4_hdmi->reset);
 	}
 
+	ret = vc4_hdmi_build_regset(vc4_hdmi, &vc4_hdmi->hdmi_regset, VC4_HDMI);
```

Comment:
Were `hdmi_regset` and `hd_regset` previously used uninitialised?

Reply:
They would have been zeroed by the kzalloc of ```vc4_hdmi``` in ```vc4_hdmi_bind```. The printing code of ```drm_print_regset32``` loops through from 0 to ```regset->nregs```, which will be 0 unless any registers have been added.

I'm relying on the same behaviour to do nothing for all the new vc5 register blocks when called on vc4.